src/components/Button/LoginButton.tsx

```diff
@@ -0,0 +1,25 @@
+import { useEffect, useState } from "react";
+import "./LoginButton.module.css";
+
+const LoginButton = () => {
```

Comment:
P1:
button의 속성들을 넘길 수 있게 작업해주시면 좋겠습니다.

```suggestion
const LoginButton = ({ type = 'button', ...props }:ButtonHTMLAttributes<HTMLButtonElement>) => {
```